src/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -174,9 +212,26 @@ def reset(self):
     def get(self):
         return self.impl.result()
 
+class _NumPyAccuracy(_Accuracy):
+
+    def __init__(self):
+        self.acc = 0.0
+        self.num_batches = 0.0
+
+    def update(self, accuracy):
+        self.acc += accuracy
+        self.num_batches += 1.0
```

Comment:
This doesn't have the same interface as the other accuracy metrics. If the last batch is a different size, it doesn't weight the samples equally.

Please look at how the MXNet and TF versions are implemented. In practice, these effectively tally (a) how many correct examples there are and (b) how many total examples there are.

Reply:
Yes I changed the interface because I can fetch accuracy from TensorFlow by calling `self.accuracy` from the same session which is running `predict` on the batch. It's more cost effective then repeating the same computation outside in `sound_classifier.py`. 

Do we really care about matching the interface of the MXNet and TF metrics? When we are going to remove them anyway.

Reply:
I'll see if I can solve the unequal weigh problem, otherwise I'll switch over to matching interface with MXNet metric and manually compute accuracy.

Reply:
We want the computation of the metric to be correct, so if we change the interfaces we should be sure we understand precisely how they work and what we need to do to preserve correctness. I'm worried that we're creating more work by trying to reinvent interfaces than it would have taken just to follow them and build upon the prior effort to sort out correct patterns

Reply:
Yes I agree. Thanks for pointing this out. Now my NumPy metric implementation matches MXNet implementation output exactly.